### ISSUE TITLE
Not available on ubuntu but needed for osc

### ISSUE THREAD

**Issue by sebix** [openSUSE/obs-service-format_spec_file#12]
osc claims to need this package (using a trusty client):

    $ osc commit
    *** Error: Package obs-service-format_spec_file is required for this operation

But this package does not exists in [official repos](http://packages.ubuntu.com/search?suite=trusty&searchon=names&keywords=obs-service-format_spec_file) (also not for other versions).

So, let's use another source: OBS itself!

https://build.opensuse.org/package/show/openSUSE:Tools/obs-service-format_spec_file

The publish flag is true, but on the download page, it's not available, same for soo: https://software.opensuse.org/package/obs-service-format_spec_file?search_term=obs-service-format_spec_file

So where to get this package? (Except building myself?)

**Comment by lethliel** [comment]
Have you added the ubuntu repo for openSUSE Tools to your /etc/apt/sources.list:
`deb http://ftp5.gwdg.de/pub/opensuse/repositories/openSUSE:/Tools/xUbuntu_14.04 ./`

I think 14.04 is trusty tahr

Then you should be able to install it.

**Comment by sebix** [comment]
1. Why is this not linked at boo and soo as explained above?
2. The corresponding key 85753AA5EEFEFDE9 for openSUSE:Tools@build.opensuse.org expired 2014-12-20 :/

So I have to build it myself...

**Comment by adrianschroeter** [comment]
1. You get the service usualy from the same source as you get osc. software.opensuse.org is not listing non maintained distros anymore.

2. the key openSUSE:Tools still runs for quite some years: osc signkey openSUSE:Tools | gpg2 --list-packets

**Comment by sebix** [comment]
1. osc is in [universe](http://packages.ubuntu.com/search?keywords=osc&searchon=names&suite=xenial&section=all), obs-service-format_spec_file not
2. There are actually two versions of this key. One expired in 2014 and the other one will expire in 2017. The first one was available on keys.gnupg.net (where I downloaded it from). I now uploaded the newer one to this server too, so this problem is now solved.
